doc/changelog.md

```diff
@@ -17,6 +17,9 @@
 * Add support for ``var`` of general observables
   [#124](https://github.com/PennyLaneAI/catalyst/pull/124)
 
+* Add support for jvp/vjp operations
+  [#98](https://github.com/PennyLaneAI/catalyst/pull/98)
+

```

Comment:
It's nice to accompany new features with an example in the changelog :)

Reply:
Hmm, example? Like a multi-line Python code snippet? I can not see any examples in the changelog, so I am confused a bit. Let me suggest a more detailed description for now, but please let me know what do you think it should like.

Reply:
If you scroll in the above snippet you should see a Python example :) Here is the commit as well: https://github.com/PennyLaneAI/catalyst/pull/96/commits/25a3ad3a6b59e8036a98f16aa619e45ab611229e

Reply:
@dime10 I can not see any examples in the head of the changelog. Given the text of your snippet, I guess that it was removed. Could you please double-check?
In general, I would also suggest keeping the Changelog brief by just putting links to docs or something like this.

Reply:
```
 * Catalyst programs can now be used inside of a larger JAX workflow which uses JIT compilation, 
   automatic differentiation, and other JAX transforms. 
   [#96](https://github.com/PennyLaneAI/catalyst/pull/96) 
  
   Note that generally Catalyst should be used to JIT the entire workflow, but sometimes users may 
   wish to delegate only the quantum part of their workflow to Catalyst and let JAX handle the rest 
   (for example due to missing a feature or compatibility issue in Catalyst). 
  
   Examples of newly supported workflows: 
  
    * JIT compilation with JAX: 
  
      ```py 
      @qjit 
      @qml.qnode(dev) 
      def circuit(x): 
          qml.RX(jnp.pi * x[0], wires=0) 
          qml.RY(x[1] ** 2, wires=0) 
          qml.RX(x[1] * x[2], wires=0) 
          return qml.probs(wires=0) 
  
      @jax.jit 
      def cost_fn(weights): 
          x = jnp.sin(weights) 
          return jnp.sum(jnp.cos(circuit(x)) ** 2) 
  
      cost_fn(jnp.array([0.1, 0.2, 0.3])) 
      ``` 
```

Reply:
> In general, I would suggest keeping the Changelog brief by just putting links to docs or something like this, but it is not much valuable for me.

Having explanations & examples in the changelog is something we follow in PennyLane.

Reply:
For reference, this is the changelog that we have currently https://github.com/PennyLaneAI/catalyst/blob/7ba3402a815956438abf33e685239da43d00392a/doc/changelog.md

Reply:
Yes the commit I linked has not been merged yet

Reply:
OK, should be it

Reply:
Yep, having richer changelogs is a deliberate decision we made on the PennyLane side. I've seen projects use both styles (brief changelogs and richer changelogs). By having the changelog include a description of the major changes or new features, it has helped us a lot in terms of:

- much more easily generating the release notes and marketing material 
- keeping track of the user-facing aspect of merged PRs

We've also gotten a lot of feedback from researchers/users of PL that the richer changelogs have been super helpful in communicating to them what a new feature actually is, and motivate them to upgrade.